.vcmrc

```diff
@@ -0,0 +1,27 @@
+{
+  "types": [
+    "feat",
+    "fix",
+    "docs",
+    "style",
+    "refactor",
+    "perf",
+    "test",
+    "build",
+    "ci",
+    "chore",
+    "revert"
+  ],
+  "scope": {
+    "required": false,
+    "allowed": ["*"],
+    "validate": false,
+    "multiple": false
+  },
+  "warnOnFail": false,
+  "maxSubjectLength": 100,
+  "subjectPattern": ".+",
+  "subjectPatternErrorMsg": "subject does not match subject pattern!",
+  "helpMessage": "Check the correct format on: https://github.com/Allan690/DevJobs-Slack-bot/wiki/Commit-Message-Format",
```

Comment:
Is this wiki available or will this PR create it?

Reply:
@Allan690  I have already created it https://github.com/Allan690/DevJobs-Slack-bot/wiki/Commit-Message-Format